tools/UpdateModules.ps1

```diff
@@ -40,6 +40,10 @@ Import-Module "$PSScriptRoot\UpdateModules.psm1"
 #################################################>
 
 
+# Constants (Scopes)
+$NetCoreScopes = @('NetCore')
+$AzureScopes = @('All', 'Latest', 'ServiceManagement', 'AzureStorage')
+$StackScopes = @('All', 'Stack')
```

Comment:
Root cause: `.psm1` has its own scope. If these variables were defined in a `.psm1` file, we cannot access them here, so `Update-Netcore` was skipped.